paper/paper.md

```diff
@@ -34,16 +34,18 @@ affiliations:
 
 ***Needs better wording***
+<!-- MSB: tried this... Not sure I got to the heart of what you meant.-->
+<!-- MSB: Could still use some love... -->
 
-(null-hypothesis significance testing, though in a Bayesian framework there is not necessarily a null-hypothesis, nor is statistical significance in its classical meaning, with associated p-values).
+(Note that null-hypothesis significance testing (NHST), though possible in a Bayesian framework, can be extended to general testing, nut just of a *null*-hypothesis, and thus does not perfectly map onto statistical significance testing in its classical meaning, with associated p-values).
 
 
-Existing R packages allow users to easily fit a large variety of models and extract, and visualize, the posterior draws. However, most of these packages only return a limited set of indices (*e.g.*, point-estimates and CI). `bayestestR` provides a comprehensive and consistent set of functions to analyze and describe a variety of models objects, including popular modeling packages such as `rstanarm` [@goodrich2018rstanarm], `brms` [@burkner2017brms] or `BayesFactor` [@morey2014bayesfactor]. Beyond computing point-estimates (mean, median or MAP estimate) and quantifying the related uncertainty (with different types of CIs), `bayestestR` focuses on implementing a null-hypothesis testing framework. By providing access to both established and exploratory indices of effect existence and significance, `bayestestR` appears as a useful tool supporting Bayesian statistics. The main functions are described below, and a full documentation is available on the [package's website](https://easystats.github.io/bayestestR).
+Existing R packages allow users to easily fit a large variety of models and extract and visualize the posterior draws. However, most of these packages only return a limited set of indices (*e.g.*, point-estimates and CIs). `bayestestR` provides a comprehensive and consistent set of functions to analyze and describe posterior distributions generated by a variety of models objects, including popular modeling packages such as `rstanarm` [@goodrich2018rstanarm], `brms` [@burkner2017brms] or `BayesFactor` [@morey2014bayesfactor]. <!-- MSB: Currently only limited to simple models. Should we mention that? --> Beyond computing point-estimates (mean, median or MAP estimate) and quantifying the related uncertainty (with different types of CIs), `bayestestR` focuses on implementing a Bayesian null-hypothesis testing framework. By providing access to both established and exploratory indices of effect existence and significance, `bayestestR` appears as a useful tool supporting Bayesian statistics. The main functions are described below, and a full documentation is available on the [package's website](https://easystats.github.io/bayestestR).
```

Comment:
> <!-- MSB: Currently only limited to simple models. Should we mention that? --> 

Don't think so since in the end, the goal is still to support them all